.github/workflows/golangci-lint.yml

```diff
@@ -24,6 +24,4 @@ jobs:
         with:
           go-version: stable
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v7
-        with:
-          version: v2.0.2
```

Comment:
@elezar What is the reason you'd prefer to go without version-locking the linter ?

Reply:
It was either this or bump the version since the newer action requires a newer version.

What benefit does locking the linter add?

Reply:
Predictability, mostly I think. Newer versions of golangci-lint has consistently started flagging things earlier version did not use to complain about.

Without a version lock (IOW without control over which version of golangci-lint is used in CI), I think it can easily happen that a PR starts failing the linter without any related changes being introduced in the PR itself, just the linter action bumping the default version of the linter being used. Whether this is a problem or a desired thing is admittedly a bit of a matter of taste.